catalog-rest-service/src/main/resources/json/schema/type/changeEvent.json

```diff
@@ -0,0 +1,67 @@
+{
+  "$id": "https://open-metadata.org/schema/type/auditLog.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "ChangeEvent",
+  "description": "This schema defines the change event type to capture the changes to entities. Entities change due to user activity, such as updating description of a dataset, changing ownership, or adding new tags. Entity also changes due to activities at the metadata sources, such as a new dataset was created, a datasets was deleted, or schema of a dataset is modified. When state of entity changes, an event is produced. These events can be used to build apps and bots that respond to the change from activities.",
+  "type": "object",
+
+  "definitions": {
+    "operation" : {
+      "method": {
+        "description": "HTTP Method used in a call.",
+        "type": "string",
+        "enum": [
+          "POST",
+          "PUT",
+          "PATCH",
+          "DELETE"
+        ]
+      },
+      "path": {
+        "description": "Requested API Path.",
+        "type": "string"
+      },
+      "responseCode": {
+        "description": "HTTP response code for the api requested.",
+        "type": "integer"
+      }
+    },
+    "eventType" :{
+      "description": "Type of event",
+      "type": "string",
+      "enum": [
+        "ENTITY_CREATED",
+        "ENTITY_UPDATED",
+        "ENTITY_DELETED",
+        "ENTITY_FOLLOWED",
+        "ENTITY_UNFOLLOWED"
+      ]
+    }
+  },
+  "properties" : {
+    "eventType" : {
+      "$ref": "#/definitions/eventType"
+    },
+    "entityType" : {
+      "description": "Entity type that changed. Use the schema of this entity to process the entity attribute.",
+      "type": "string"
+    },
+    "entity" : {
+      "description": "Entity that changed."
+    },
+    "userName": {
+      "description": "Name of the user whose activity resulted in the change.",
+      "type": "string"
+    },
+    "operation" : {
+      "description": "API operation that was result of user activity resulted in the change.",
+      "$ref": "#/definitions/operation"
+    },
+    "dateTime": {
```

Comment:
Should we consider standardizing on timestamp here (and other possible schemas)?
And convert to dateTime on any external facing user interface with local time zone?

Reply:
@mithmatt , dateTime is already standardized no?
```
    "dateTime": {
      "description": "Date and time when the change was made.",
      "$ref": "basic.json#/definitions/dateTime"
    }
```

And you look at the referenced type `dateTime` from `basic.json`:
```
    "dateTime": {
      "description": "Date and time in ISO 8601 format. Example - '2018-11-13T20:20:39+00:00'.",
      "type": "string",
      "format": "date-time"
    },
```

Maybe I misunderstood your comment?

Reply:
Yes we have dateTime standardized. 

What I intended to ask:
For all entities in backend (including db storage), is it more beneficial to use epoch Millisecond/seconds instead of ISO 8601 format?

Reasoning:
1. Orgs that adopt OpenMetadata system would have global employees. Make the UI do less work to convert to browser time zone of employee. 

Zone change conversion from 1636182329 to  '2018-11-13T23:20:39+03:00'   has less steps than doing
'2018-11-13T20:20:39+00:00' --> 1636182329 --> '2018-11-13T23:20:39+03:00' (string --> timestamp in integer and increment browser's zone offset --> string with different zone)
(These conversions happen internally within the library we end up using)


2. Makes comparisons simpler/faster 
Eg: sorting change events by epoch Milliseconds... 
comparisons can easily be performed as integers/numbers  1636191094 > 1636182094
whereas ISO 8601 means comparing strings